tests/format_tests.cpp

```diff
@@ -61,9 +61,9 @@ TEST_CASE("Simple SF2 Load", "[formats]")
             rms = sqrt(rms);
             rmses.push_back(rms);
         }
-        REQUIRE(rmses[0] == Approx(19.88632).margin(1e-3));
-        REQUIRE(rmses[1] == Approx(20.96383).margin(1e-3));
-        REQUIRE(rmses[2] == Approx(20.93767).margin(1e-3));
+        REQUIRE(rmses[0] == Approx(15.14437).margin(1e-3));
+        REQUIRE(rmses[1] == Approx(18.16663).margin(1e-3));
+        REQUIRE(rmses[2] == Approx(18.72027).margin(1e-3));
```

Comment:
Why did these change? That seems wrong no?

Reply:
I think these changed because of the change to `block_size`.  I hadn't observed any detrimental effects on smoke testing the implementation with various audio configurations and buffer sizes.

Reply:
Right so a fix I prefer (and I know it is pedantic but it keeps the test the same) is rather than lower the RMS increase ihe iterations. The test below indicates runs in seconds so is basically fine. In this case in the loop above if we change the bound from 100 to 200, and the trigger points from 30/70 to 60/140, then these values should be unchanged *and* the change to the test makes more sense (“block size halved so we doubled iteration” vs “we randomly changed these RMS values”). See what I mean? Tiny change but one I think is better

Reply:
As per 2d8a1e3, doesn't look like the old RMS values can be used despite adjusting POIs to be double that of previously.